k8s/baka/templates/statefulset.yaml

```diff
@@ -28,6 +29,45 @@ spec:
       serviceAccountName: {{ include "baka.serviceAccountName" . }}
       securityContext:
         {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      initContainers:
+        - name: download
+          image: busybox:1.28
+          command:
+            - wget
+            - "-O"
+            - "/workdir/index.html"
+            - "http://info.cern.ch/"
+          volumeMounts:
+            - name: app-python-workdir-volume
+              mountPath: /workdir
+        - name: hello1
+          image: busybox:1.28
+          command:
+            - "sh"
+            - "-c"
+            - "echo 'Hello from init container 1' > /workdir/hello.txt"
+          volumeMounts:
+            - name: app-python-workdir-volume
+              mountPath: /workdir
+        -   name: hello2
```

Comment:
not sure why you need those but let them be ![](https://steamcommunity-a.akamaihd.net/economy/emoticon/:ohh_yeah:)